js/src/sdk/models/triggers.ts

```diff
@@ -60,7 +60,7 @@ export class Triggers {
             });
             return response.data as {status: string, triggerId: string};
         } catch (error) {
-            throw CEG.handleError(error);
+            throw CEG.handleAllError(error);
```

Comment:
The code contains a bug where throw CEG.handleAllError(error); is used in multiple catch blocks. However, CEG.handleAllError(error) itself throws an exception internally, which means that the throw statement attempts to throw undefined. This leads to runtime errors or unintended behavior, as the original error may be lost or result in unhandled exceptions. The throw keyword should be removed from these calls to allow CEG.handleAllError(error); to execute correctly. Additionally, within the handleAllError method, if isResponseNotPresent is true, the call to handleNonResponseAxiosError(error) always throws an error, making any subsequent code unreachable. This indicates a logical error that could lead to unexpected behavior.


Please check all the places.

Reply:
Instead throw and throw. Added `CEG.handleAllError(error)` with argument shouldThrow, otherwise it returns error object.